templates/shared/repo_search.tmpl

```diff
@@ -0,0 +1,75 @@
+<div class="ui secondary filter menu gt-ac gt-mx-0">
+	<form class="ui form ignore-dirty gt-f1">
+		<input type="hidden" name="sort" value="{{$.SortType}}">
+		<input type="hidden" name="language" value="{{$.Language}}">
+		<div class="ui fluid action input">
+			{{template "shared/searchinput" dict "Value" .Keyword}}
+			{{if .PageIsExploreRepositories}}
+				<input type="hidden" name="only_show_relevant" value="{{.OnlyShowRelevant}}">
+			{{else if .TabName}}
+				<input type="hidden" name="tab" value="{{.TabName}}">
+			{{end}}
+			<button class="ui primary button">{{ctx.Locale.Tr "explore.search"}}</button>
+		</div>
+	</form>
+	{{$tabQuery := printf "tab=%s&" .TabName}}
+	{{if not .TabName}}{{$tabQuery = ""}}{{end}}
+	{{$languageQuery := printf "language=%s&" .Language}}
+	{{if not .TabName}}{{$languageQuery = ""}}{{end}}
+	{{$queryParams := printf "%s%sq=%s" $tabQuery $languageQuery .Keyword}}
```

Comment:
At first glance, many things need to be correctly escaped.

Reply:
should be better

Reply:
> should be better now

Hmm, it doesn't look right. The manual escaping problem in Golang template is quite complex. Will take a look later if I would get enough time.

(you could also refer to other "printf" usages to see the details)

Reply:
I don't think that any escaping is required, because it was not escaped before and is from backend and can be trusted

Reply:
I hope its ok now